franck-bereket.html

```diff
@@ -0,0 +1,11 @@
+<main id="main">
+    <link rel="stylesheet" href="styles.css">
+    <title id="title">Dr Norman</title>
+    <div id="img-div">
+      <img src="" id="image" >
+  
+        <figcaption id="img-caption">not empty</figcaption>
+      <tribute id="tribute-info">text info</tribute>
+      <a href="#" id="tribute-link"target="_blank">tribute information</a>
+    </div>
```

Comment:
I think it would be better, in order to make your code well structured, you should think about using semantic tags to better structure your code, Note that the content displayable in a browser must be in the body tag because, without the latter, nothing will be displayed each time we want to try your code in a browser